pkg/kudoctl/packages/types.go

```diff
@@ -37,6 +37,12 @@ type Parameter struct {
 	Trigger     string                `json:"trigger,omitempty"`
 	Type        kudoapi.ParameterType `json:"type,omitempty"`
 	Immutable   *bool                 `json:"immutable,omitempty"`
+	Enum        *[]string             `json:"enum,omitempty"`
+
+	// The following fields are descriptive only and are not used in the OperatorVersion
```

Comment:
I did not really mean what does the descriptive mean here, maybe it's just my problem buuut ... could we maybe say something like that these are prperties that live only on the package level and are used for additional data during the installation and are not necessary server-side? Something along those lines... I actually had to go back to KEP to understand this better